code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -667,12 +675,13 @@ var/global/list/robot_modules = list(
 	RG.add_reagent(/decl/reagent/polysomnine/beer2, 50)
 	emag.name = "Mickey Finn's Special Brew"
 
-/obj/item/robot_module/clerical/general
+/obj/item/robot_module/service/clerical
 	name = "clerical robot module"
 	channels = list(CHANNEL_SUPPLY = TRUE, CHANNEL_COMMAND = TRUE)
 	networks = list(NETWORK_MINE)
+	specialized_access_types = list(/datum/job/janitor, /datum/job/librarian, /datum/job/hangar_tech)
```

Comment:
clerical module shouldn't have any special access.

Reply:
Hmm... it needs some sort of access to help fill out forms. Like a general department access without specifics. I am not sure which role works best for it, but leaving it with just assistant access feels weird

Reply:
i recommend directly defining necessary accesses.